### PR TITLE
GKE provider: Add cluster-ipv4-cidr and arbitrary flags

### PR DESCRIPTION
**Pull request by zmerlynn**
cc @roberthbailey @a-robinson @cjcullen

---

This change is [Reviewable](http://reviewable.k8s.io/reviews/kubernetes/kubernetes/25437)

### CODE REVIEW

File: cluster/gke/util.sh [hunk 1]
```diff
@@ -166,6 +168,12 @@ function kube-up() {
     create_args+=("--additional-zones=${ADDITIONAL_ZONES}")
   fi
 
+  if [[ ! -z "${CLUSTER_IP_RANGE:-}" ]]; then
+    create_args+=("--cluster-ipv4-cidr=${CLUSTER_IP_RANGE}")
+  fi
+
+  create_args+=( ${GKE_CREATE_FLAGS:-} )
```
Comment (a-robinson) [inline cluster/gke/util.sh]: Can you explain why we don't want to put ${GKE_CREATE_FLAGS:-} in quotes?

**Comment by a-robinson** [comment]
LGTM

File: cluster/gke/util.sh (hunk 1 above)
Comment (zmerlynn) [inline cluster/gke/util.sh]: It's intentional - it implicitly splits across spaces and dumps it in the array, e.g.:

```
$ LIST="foo bar blat"
$ ARR=( ${LIST} )
$ echo ${ARR[2]}
blat
```

Comment (a-robinson) [inline cluster/gke/util.sh]: Thanks! I figured it was intentional, just wanted a free bash lesson from you :)

**Pull request merged**